regcore/urls.py

```diff
@@ -18,5 +19,6 @@
         path("<date>/title/<title>", EffectivePartsView.as_view()),
         path("<date>/title/<title>/part/<name>", EffectivePartView.as_view()),
         path("<date>/title/<title>/part/<name>/toc", EffectivePartTocView.as_view()),
+        path("title/<title>/existing", ExistingPartsView.as_view())
```

Comment:
Recommend comma after this line for cleaner future diffs

Reply:
added